### PR TITLE
Adds more witty comments.

### PR DESCRIPTION
**Pull request by YTG1234**
Those are also configurable because yes.

The category is WIP because this is not gameplay-related.

### CODE REVIEW

File: src/main/java/io/github/protonmc/proton/mixin/tweaks/CrashReportMixin.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@Mixin(CrashReport.class)
+public abstract class CrashReportMixin {
+    @ModifyVariable(method = "generateWittyComment()Ljava/lang/String;", at = @At(value = "STORE", ordinal = 0), ordinal = 0)
+    @FromModule(MoreWittyMessages.class)
+    private static String[] addWittyComments(String[] orig) {
+        if (ModuleManager.getInstance().isModuleEnabled(MoreWittyMessages.class)) {
+            String[] newStrings = new String[orig.length + wittyMessages.length];
+            System.arraycopy(orig, 0, newStrings, 0, orig.length);
+            System.arraycopy(wittyMessages, 0, newStrings, orig.length - 1, wittyMessages.length);
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/mixin/tweaks/CrashReportMixin.java:21]: Yes, I did it in a very hacky way

File: src/main/java/io/github/protonmc/proton/module/tweaks/MoreWittyMessages.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package io.github.protonmc.proton.module.tweaks;
+
+import io.github.protonmc.proton.Proton;
+import io.github.protonmc.proton.base.module.ProtonModule;
+import io.github.protonmc.tiny_config.Configurable;
+
+public class MoreWittyMessages extends ProtonModule {
+    @Configurable
+    public static String[] wittyMessages = {
+            "OOPSIE WOOPSIE!! Uwu We made a fucky wucky!! A wittle fucko boingo! The code monkeys at our headquarters are working VEWY HAWD to fix this!"
```
Comment (YTG1234) [inline src/main/java/io/github/protonmc/proton/module/tweaks/MoreWittyMessages.java:10]: OG message is the default message

File: src/main/java/io/github/protonmc/proton/Proton.java [hunk 3]
```diff
@@ -14,21 +14,20 @@
 
 public class Proton implements ModInitializer {
```
Comment (hYdos) [inline src/main/java/io/github/protonmc/proton/Proton.java]: random space here?

File: src/main/java/io/github/protonmc/proton/mixin/tweaks/CrashReportMixin.java (hunk 1 above)
Comment (hYdos) [reply src/main/java/io/github/protonmc/proton/mixin/tweaks/CrashReportMixin.java:21]: CONCERN

File: src/main/resources/proton.accessWidener [hunk 4]
```diff
@@ -1,4 +1,4 @@
 accessWidener   v1  named
 
 # Methods
-accessible  method net/minecraft/block/Blocks   never   (Lnet/minecraft/block/BlockState;Lnet/minecraft/world/BlockView;Lnet/minecraft/util/math/BlockPos;)Z
\ No newline at end of file
+accessible  method  net/minecraft/block/Blocks  never   (Lnet/minecraft/block/BlockState;Lnet/minecraft/world/BlockView;Lnet/minecraft/util/math/BlockPos;)Z
```
Comment (hYdos) [inline src/main/resources/proton.accessWidener:4]: tabs are better imo

**Review by hYdos — CHANGES_REQUESTED** [review]
e

File: src/main/resources/proton.accessWidener (hunk 4 above)
Comment (YTG1234) [reply src/main/resources/proton.accessWidener:4]: I did tabs, I think IntelliJ made them into spaces

File: src/main/java/io/github/protonmc/proton/Proton.java (hunk 3 above)
Comment (YTG1234) [reply src/main/java/io/github/protonmc/proton/Proton.java]: Fixed

File: src/main/resources/proton.accessWidener (hunk 4 above)
Comment (hYdos) [reply src/main/resources/proton.accessWidener:4]: oh concern. we need checkstle to fix that. probably out of the scope of this pr though.

**Review by hYdos — APPROVED** [review]

**Review by namebee — CHANGES_REQUESTED** [review]
Undo changes to Proton.java, Unnecessary and annoying on small screens.

**Comment by YTG1234** [comment]
> Undo changes to Proton.java, Unnecessary and annoying on small screens.

Fixed

**Review by namebee — APPROVED** [review]
This is absolutely horrifying, I love it.

**Review by dzwdz — APPROVED** [review]
oh no

**Comment by YTG1234** [comment]
yeah so I like to make everything configurable, that's why the code is so hacky.

**Pull request merged**